pkg/systemd/services/services.jsx

```diff
@@ -245,10 +244,10 @@ class ServicesPageBody extends React.Component {
                     .catch(error => {
                         if (error.name != "org.freedesktop.systemd1.AlreadySubscribed" &&
                         error.name != "org.freedesktop.DBus.Error.FileExists")
-                            this.setState({ error: cockpit.format(_("Subscribing to systemd signals failed: $0"), error.toString()), loadingUnits: false });
+                            this.setState({ error: cockpit.format(_("Subscribing to systemd signals failed: $0"), error.toString()) });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18005-20221207-164851-4022857c-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)